tasks.sh

```diff
@@ -97,6 +98,7 @@ elif [ "${TARGET_ARCH}" = "arm64" ]; then
   : ${PROOT_ARGS:="-q qemu-aarch64-static"}
   : ${SYZKALLER_TARGETARCH:="arm4"}
   : ${ROOT_MNT:="/dev/vda"}
+  : ${CROSS_COMPILE:="aarch64-linux-gnu-"}
```

Comment:
Mh I'm surprised this is necessary. When using `LLVM=1`, I don't need to every provide a `CROSS_COMPILE` flag. I intentionally left that one out originally because I didn't want to have to pick a target triplet here. My experience is that there are bazillions of variants of "arm64 linux" toolchain triplet and they are wrong more often than not. Are you maybe using downstream changes to use GCC ? GCC needs that but not LLVM in my experience.

Or do you have further evidences that this is an issue ? Eg: reproducing steps etc.